app/admin/dashboard.rb

```diff
@@ -3,31 +3,28 @@
   menu priority: 1, label: proc{ I18n.t("active_admin.dashboard") }
 
   content title: proc{ I18n.t("active_admin.dashboard") } do
-    div class: "blank_slate_container", id: "dashboard_default_message" do
-      span class: "blank_slate" do
-        span I18n.t("active_admin.dashboard_welcome.welcome")
-        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+    columns do
+      column do
+        panel "Recent Books" do
+          ul do
+            Book.last(10).map do |book|
+              li link_to(book.title, admin_book_path(book))
+            end
+          end
+          strong link_to "View All Books", admin_books_path
+        end
       end
-    end
-
-    # Here is an example of a simple dashboard with columns and panels.
-    #
-    # columns do
-    #   column do
-    #     panel "Recent Posts" do
-    #       ul do
-    #         Post.recent(5).map do |post|
-    #           li link_to(post.title, admin_post_path(post))
-    #         end
-    #       end
-    #     end
-    #   end
 
-    #   column do
-    #     panel "Info" do
-    #       para "Welcome to ActiveAdmin."
-    #     end
-    #   end
-    # end
-  end # content
+      column do
+        panel "Recent Orders" do
+          ul do
+            Order.last(10).map do |order|
```

Comment:
I would probably put something here to enforce an order by created_at
 Add a line note